examples/java-agent/src/main/java/example/HelloWorld.java

```diff
@@ -17,9 +17,14 @@
 package example;
 
 import static spark.Spark.get;
+import static spark.Spark.port;
```

Comment:
The static import seems rather non-google style.
Should we be using `Spark.port(...)`, `Spark.get(...)`

Reply:
It is SparkJava style though.  For example: http://sparkjava.com and http://sparkjava.com/documentation#port